### PR TITLE
Initial API mock

### PR DESCRIPTION
**Pull request by srri-zz**
Initial implementation of django rest framework

### CODE REVIEW

File: dispatch/api/serializers.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from rest_framework import serializers
+
+#class ArticleSerializer(serializers.HyperlinkedModelSerializer):
+#   class Meta:
+#        model = Article
+#        fields = ('headline')
+
+class UserSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        User = get_user_model()
```
Comment (psiemens) [inline dispatch/api/serializers.py]: I don't think User should be inside Meta

Comment (srri-zz) [inline dispatch/api/serializers.py]: Well I disagree :-1:

Comment (srri-zz) [inline dispatch/api/serializers.py]: But i'll move it

Comment (psiemens) [inline dispatch/api/serializers.py]: :imp:

**Comment by psiemens** [comment]
:+1:

**Pull request merged**